person.rb

```diff
@@ -12,9 +18,19 @@ def can_use_services?
     of_age? || @parent_permission
   end
 
+  def correct_name
+    @name
+  end
+
   private
 
   def of_age?
     @age >= 18
   end
 end
+
+person = Person.new(12, name: 'esmat')
+capitalize_person = Capitalize_decorator.new(person)
+puts capitalize_person.correct_name
+capitalize_trimmed_person = Trimmer_decorator.new(capitalize_person)
+puts capitalize_trimmed_person
```

Comment:
- [ ] Please ensure that you are calling the correct methods to avoid `NameError` as a result of the method you are calling being unitialized:
- Call `CapitalizeDecorator` instead of `Capitalize_decorator`
- Call `TrimmerDecorator` instead of `Trimmer_decorator`
![image](https://github.com/Esmatullah12/school-library/assets/46839548/2dbb858b-4c86-4664-be81-a24453a77dda)